onnx2pytorch/operations/pad.py

```diff
@@ -12,6 +12,12 @@ def __init__(self, mode="constant", padding=None):
     def forward(self, input, pads=None, value=0):
         if self.padding is not None:
             pads = self.padding
-        elif pads is None:
+        elif pads is not None:
+            pass
```

Comment:
Why this change?

Reply:
Just some debugging cruft which snuck through -- now removed.